### PR TITLE
Create fra.yml

### PR DESCRIPTION
**Pull request by Bqleine**

### CODE REVIEW

File: resources/translations/fra.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+codes:
+- fr_FR
+
```
Comment (Endermanbugzjfc) [inline resources/translations/fra.yml:4]:
```suggestion
- fr_CA

```

**Review by Endermanbugzjfc — APPROVED** [review]
*unhappy canada french speaker noises (not me)*

**Comment by Bqleine** [comment]
You can add it for the québec gamers

**Comment by Endermanbugzjfc** [comment]
Tysm.

**Pull request merged**